build_qrc.py

```diff
@@ -0,0 +1,31 @@
+import os
+import sys
+
+qrc_file_path = "/Users/iseyuki/pyqtTodo/resources.qrc"
```

Comment:
Please change to relative paths. As it stands, it is not runnable by anyone but you.

Reply:
I just updated absolute paths to relative paths